adafruit_dash_display.py

```diff
@@ -145,7 +163,7 @@ def __init__(self, display, io, nav):
         self.display.show(self.splash)
 
     def simple_text_callback(
-        self, client, feed_id, message
+        self, client: Any, feed_id: str, message: str
```

Comment:
I think that all of these places where `client` is typed as `Any` it is expected to be the `IO_MQTT` type. I will try to test this out on a device and add some logging output to confirm

Reply:
I confirmed that these are IO_MQTT by adding some `print(type(client))` statements in a few of these functions. The latest commit changes type declaration for client `IO_MQTT`

Reply:
Thank you for looking into this!